### PR TITLE
Jordan/1910 clipboard

### LINKED ISSUE #1910: Bug in "Copied" for the address
UI Version: latest web version

Description:

MacOS: works in Chrome, doesn't show up in Safari, works in Firefox but there is bad formatting

Safari does not copy or show that it is copied:
screen shot 2019-01-30 at 12 16 47

Mozilla works, but see the formatting
screen shot 2019-01-30 at 12 28 21

### PR DESCRIPTION
**Pull request by jbibla**
Closes #1910 

**Description:**

- added clipboard directive for cross browser support
- also fixed cross browser styling issue with tooltips
- deleted TmBtnCopy, AnchorCopy, TmLiCopy, and TmModalReceive

Thank you! 🚀

---

For contributor:

- [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: test/unit/specs/components/common/ShortBech32.spec.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+localVue.directive(`clipboard`, VueClipboard)
+localVue.directive(`tooltip`, () => {})
+
 describe(`ShortBech32`, () => {
   let wrapper
-  const instance = setup()
 
   beforeEach(() => {
-    const test = instance.mount(ShortBech32, {
+    wrapper = mount(ShortBech32, {
```
Comment (sabau) [inline test/unit/specs/components/common/ShortBech32.spec.js]: I would try to go on `shallowMount`

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - [\#1958](https://github.com/cosmos/voyager/issues/1958) Kill all processes if one dies to not have dangling processes in development setups @faboweb
 - Fix development warning overlaying the UI too much @faboweb
+- clipboard works in all browsers @jbibla
```
Comment (sabau) [inline CHANGELOG.md]: I think this suold go in the unreleased section right?

File: test/unit/specs/components/common/ShortBech32.spec.js [hunk 3]
```diff
@@ -35,7 +38,17 @@ describe(`ShortBech32`, () => {
...
+    jest.useFakeTimers()
+    wrapper.vm.onCopy()
+    expect(wrapper.vm.copySuccess).toBe(true)
+
+    jest.runAllTimers()
+    expect(wrapper.vm.copySuccess).toBe(false)
+  })
+
+  // not sure how to test the v-clipboard directive events
+  xit(`clicking copy copies the address`, () => {
```
Comment (fedekunze) [inline test/unit/specs/components/common/ShortBech32.spec.js]: add `TODO:` ?

File: test/unit/specs/components/common/ShortBech32.spec.js (hunk 1 above)
Comment (jbibla) [reply test/unit/specs/components/common/ShortBech32.spec.js]: good call! will do!

File: CHANGELOG.md (hunk 2 above)
Comment (jbibla) [reply CHANGELOG.md]: yes!

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,10 +12,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [\#2000](https://github.com/cosmos/voyager/issues/2000) Add distribution endpoints support @fedekunze
 
 ### Changed
+
 - [\#1936](https://github.com/cosmos/voyager/issues/1936) remove `prettier` to avoid overriding `eslint:vue` rules @sabau
 
 ### Fixed
 
+- [\#1910](https://github.com/cosmos/voyager/issues/1910)clipboard works in all browsers @jbibla
```
Comment (fedekunze) [inline CHANGELOG.md]:
```suggestion
- [\#1910](https://github.com/cosmos/voyager/issues/1910) clipboard works in all browsers @jbibla
```

**Pull request merged**